packages/dds/tree/src/test/snapshots/testTrees.ts

```diff
@@ -52,6 +53,7 @@ const rootNode: UpPath = {
 
 const factory = new SharedTreeFactory({
 	jsonValidator: typeboxValidator,
+	treeEncodeType: TreeCompressionStrategy.Uncompressed,
```

Comment:
Could we remove this line, as the `defaultSharedTreeOptions` already sets the `treeEncodeType` to `TreeCompressionStrategy.Uncompressed`? But maybe it might be better to leave it as is if we decide to change the default `treeEncodeType` in the future.

Reply:
Yes. I also figured changing this should be separate from changing the default, but really either approach would be ok.